wa/instruments/delay.py

```diff
@@ -0,0 +1,231 @@
+#    Copyright 2013-2018 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+#pylint: disable=W0613,E1101,E0203,W0201
+import time
+
+from wa import Instrument, Parameter
+from wa.framework.exception import ConfigError, InstrumentError
+from wa.framework.instrument import very_slow
```

Comment:
As of https://github.com/ARM-software/workload-automation/commit/864ded60ae40c621ccab4cf6f9721f81fa02ee34, should use `extremely_slow` thoughout.